src/modules/10-common/components/ResourceCenter/SubmitTicketModal/SubmitTicketModalSteps/Utils.tsx

```diff
@@ -1,6 +1,6 @@
 export const getComponentsFromModule = (module: string): Array<{ label: string; value: string }> => {
```

Comment:
```suggestion
export const getComponentsFromModule = (module: ModuleName): Array<{ label: string; value: string }> => {
```
Adding proper type to module should have not caused this issue.